### PR TITLE
feat(chat) add support for sending and receiving reactions 

### PR DESCRIPTION
**Pull request by he-patrick**
- build and send xml package in sendReaction
- process received reactions in onMessage
- emit event for frontend to update UI

### CODE REVIEW

File: JitsiConference.js
Comment (he-patrick) [inline JitsiConference.js:1]: Can I remove receiverId since the reactions that are attached to private messages will not be visible to others anyways?

Comment (saghul) [reply JitsiConference.js:1]: Not really, because we want the sender to decide to whom a reaction should be sent.

**Review by saghul — APPROVED** [review]

**Comment by saghul** [comment]
Jenkins please test this please.

**Comment by saghul** [comment]
@he-patrick I noticed you put the PR as a draft. Is there something missing.

**Comment by he-patrick** [comment]
Should be ready now.

**Review by saghul — APPROVED** [review]

**Comment by saghul** [comment]
Can you please rebase?

**Comment by he-patrick** [comment]
I'm seeing that it's already up to date with master

**Pull request merged**